vehicle/mkvi/software/brakes/bspd.h

```diff
@@ -0,0 +1,38 @@
+#include "libs/adc/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+
+//////////////////////// GPIO - DIGITAL ////////////////////////
+//These are digital outputs for avr-controlled LED outputs
+gpio_t MOTOR_5KW_LED = PB0;
+gpio_t BRAKE_LL_LED = PB1;
+gpio_t BSPD_TRIP_LED = PB2;
+
+//Monitor Pins connected to the logic-level (LL) side of the LSDs
+gpio_t BSPD_LL = PB3;
+gpio_t BRAKELIGHT_LL = PB4;
+
+//Digital sense pin for 5kW motor "on" state
+gpio_t MOTOR_CURRENT_SENSE = PB5;
+
+//Input for shutdown sense line
+//gpio_t BSPD_SHUTDOWN_SENSE = PC0;
+
+//////////////////////// GPIO - ANALOG /////////////////////////
+adc_pin_e BRAKE_PRESSURE_SENSE = ADC7;
+
+//Timer config
+void timer0_callback(void);
+
+timer_cfg_s timer0_cfg = {
```

Comment:
Can you add a comment here saying what the timer configuration does? (ie run the 100hz clock for the main loop)
It's something isn't immediately obvious to a reader who isn't already acquainted with the common timer configs we use on the team :)